driver-core/src/main/java/com/datastax/driver/core/DataType.java

```diff
@@ -188,7 +188,7 @@ static DataType decode(ByteBuf buffer, ProtocolVersion protocolVersion, CodecReg
                     DataType fieldType = decode(buffer, protocolVersion, codecRegistry);
                     fields.add(new UserType.Field(fieldName, fieldType));
                 }
-                return new UserType(keyspace, type, fields, protocolVersion, codecRegistry);
+                return new UserType(keyspace, type, true, fields, protocolVersion, codecRegistry);
```

Comment:
Well actually that's a typo, I intended to return false (see my comment in the PR description). I know this changes the behavior but it's more logical to return false, and I doubt anyone relied on that in the past. I'll add a test and find a place to document it.